config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/users/:user_id/posts", to: "posts#index"
+  get "/users/:user_id/posts/:id", to: "posts#show"
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
```

Comment:
- [ ] [OPTIONAL] To make your routes cleaner. Kindly write the routes in that way
```ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
 end
```